presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestingMySqlServer.java

```diff
@@ -22,12 +22,17 @@
 
 import static java.lang.String.format;
 
-public class TestingMySqlServer
+public final class TestingMySqlServer
         extends MySQLContainer<TestingMySqlServer>
 {
     public TestingMySqlServer()
     {
-        super("mysql:8.0.12");
+        this("8.0.12");
```

Comment:
i'd keep image name and version together (`"mysql:8.0.12"`) -- no much drawback but a lot more flexible.

Also, we should run tests with Maria DB (follow up)